### PR TITLE
fix(vscode): Set default path when opt-out

### PR DESCRIPTION
**Pull request by ccastrotrejo**
### Main Code Changes
- Set the system wide default binary dependencies path when user opts out

AB#25634898

### CODE REVIEW

File: apps/vs-code-designer/src/app/utils/binaries.ts [hunk 1]
```diff
@@ -409,7 +413,7 @@ export async function promptInstallBinariesOption(context: IActionContext) {
   if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
     const workspace = await getWorkspaceFolder(context);
     const projectPath = await tryGetFunctionProjectRoot(context, workspace);
-    const message = localize('useBinaries', 'Always validate and install the latest dependency binaries at launch');
+    const message = localize('useBinaries', 'Always validate and install the latest dependency binaries at launch TEST');
```
Comment (Eric-B-Wu) [inline apps/vs-code-designer/src/app/utils/binaries.ts:416]: the TEST part should be removed, right?

**Review by Eric-B-Wu — APPROVED** [review]

File: apps/vs-code-designer/src/package.json [hunk 2]
```diff
@@ -847,7 +847,7 @@
           },
           "azureLogicAppsStandard.autoBinariesInstallation": {
             "type": "boolean",
-            "description": "Start using binaries installation",
+            "description": "Start using binaries installation Test",
```
Comment (Eric-B-Wu) [inline apps/vs-code-designer/src/package.json:850]: should be removed?

Comment (ccastrotrejo) [inline apps/vs-code-designer/src/package.json:850]: @ecfan Could you please review this string?
This is the text for the configuration setting that will tell the user if they select the auto validation/installation of the binary dependencies

File: apps/vs-code-designer/src/app/utils/binaries.ts (hunk 1 above)
Comment (ccastrotrejo) [reply apps/vs-code-designer/src/app/utils/binaries.ts:416]: Yep, just made the change so Esther can review the strings 😅

File: apps/vs-code-designer/src/package.json (hunk 2 above)
Comment (ccastrotrejo) [reply apps/vs-code-designer/src/package.json:850]: Yep, just made the change so Esther can review the strings 😅

File: apps/vs-code-designer/src/app/utils/binaries.ts (hunk 1 above)
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/utils/binaries.ts:416]: @ecfan Could you please review this string?
This is the text for the warning message that will tell the user if they select the auto validation/installation of the binary dependencies
Screenshot 2023-10-30 at 9 41 12 AM

**Comment by ecfan** [comment]
#assign:ecfan

File: apps/vs-code-designer/src/package.json (hunk 2 above)
Comment (ecfan) [inline apps/vs-code-designer/src/package.json:850]: We don't need "Test", correct?

```suggestion
            "description": "Start using binaries installation",
```

File: apps/vs-code-designer/src/app/utils/binaries.ts (hunk 1 above)
Comment (ecfan) [inline apps/vs-code-designer/src/app/utils/binaries.ts:416]:
```suggestion
    const message = localize('useBinaries', 'Always validate and install the latest dependency binaries at launch');
```

File: apps/vs-code-designer/src/package.json (hunk 2 above)
Comment (ccastrotrejo) [reply apps/vs-code-designer/src/package.json:850]: Yes, no need for the TEST. Sorry I just added for the PR to be able to mark where I wanted you to review the string. 
Here it is how its going to look like
Screenshot 2023-10-30 at 11 01 40 AM

**Comment by ecfan** [comment]
[like]  Esther Fan reacted to your message:
________________________________

**Review by ecfan — APPROVED** [review]

**Pull request merged**